src/gluonts/ext/r_forecast/_univariate_predictor.py

```diff
@@ -95,79 +99,59 @@ def __init__(
 
         self.method_name = method_name
         self._r_method = self._robjects.r[method_name]
+
         self.params = {
             "prediction_length": self.prediction_length,
-            "output_types": ["samples"],
             "frequency": self.period,
         }
 
-        if params is not None:
-            self.params.update(params)
+        if self.method_name in UNIVARIATE_POINT_FORECAST_METHODS:
+            self.params["output_types"] = ["mean"]
+        elif self.method_name in UNIVARIATE_QUANTILE_FORECAST_METHODS:
+            self.params["output_types"] = ["quantiles", "mean"]
```

Comment:
Quick question here about whether you always get "mean" forecasts: if I sent `params["output_types"] = ["quantiles"]` then `self.params` would be updated with that (line 125) and you don't get mean forecasts from the underlying method. But you are expecting the "mean" key in `forecast_dict` in the first line of function `_forecast_dict_to_obj`?

Reply:
good point :-) let me fix it

Reply:
Fixed